src/FormBuilder/resources/js/form-builder/src/components/sidebar/panels/FormPrepublishPanel.tsx

```diff
@@ -0,0 +1,152 @@
+     handleClose
+ }: FormPrepublishPanelProps) {
+
+    const {
+        settings: {formTitle, formStatus, newFormStatus},
+    } = useFormState();
+    const dispatch = useFormStateDispatch();
+    const {
+        formPage: {permalink},
+    } = getWindowData();
+
+    const isPrivate = () => {
+        if (newFormStatus) {
+            return 'private' === newFormStatus;
+        }
+
+        return 'private' === formStatus;
+    }
+
+    const permalinkField: RefObject<HTMLInputElement> = createRef();
+
+    return createPortal(
+        <div
+            className={cx('givewp-next-gen-prepublish-panel', {'givewp-next-gen-prepublish-panel__animate': !isPublished})}>
+            {isSaving ? (
+                <>
+                    <div className="givewp-next-gen-prepublish-panel__header">
+                        <div className="givewp-next-gen-prepublish-panel__header-actions">
+                            <Button
+                                variant="primary"
+                                isBusy={true}
+                                isPressed={true}
+                            >
+                                {__('Publishing', 'give')}
+                            </Button>
+                        </div>
+                    </div>
+                    <div className="givewp-next-gen-prepublish-panel__spinner">
+                        <Spinner />
+                    </div>
+                </>
+            ) : (
+                <>
+                    {isPublished ? (
+                        <>
+                            <div className="givewp-next-gen-prepublish-panel__header">
+                                <div className="givewp-next-gen-prepublish-panel__header-actions">
+                                    <Button
+                                        variant="secondary"
+                                        onClick={handleClose}
+                                    >
+                                        {__('Close panel', 'give')}
+                                    </Button>
+                                </div>
+                            </div>
+
+                            <div className="givewp-next-gen-prepublish-panel__content_publish">
+                                <strong>
+                                    <a href={permalink}
+                                       target="_blank">{formTitle}
+                                    </a> {__('is now live', 'give')}.
+                                </strong>
+                            </div>
+
+                            <PanelBody title={__("What's next?", 'give')} initialOpen={true}>
+                                <PanelRow className="givewp-next-gen-prepublish-panel_link">
+                                    <span>
+                                        {__('PAGE ADDRESS', 'give')}
+                                    </span>
+                                    <span>
+                                        <a
```

Comment:
This should probably be a button since it is a clickable element, as opposed to using an href for the styles. The Button component has a tertiary style which looks like a link. Also, it has support for an icon component built in.

Reply:
Resolved https://github.com/impress-org/givewp/pull/7050/commits/ac54f4604de4b4fd2291dcaa9255d34f2ab70cd0